libs/utils/utils.js

```diff
@@ -1074,8 +1075,7 @@ export function scrollToHashedElement(hash) {
 }
 
 export async function loadDeferred(area, blocks, config) {
-  const event = new Event(MILO_EVENTS.DEFERRED);
-  area.dispatchEvent(event);
+  area.dispatchEvent(new Event(MILO_EVENTS.DEFERRED));
```

Comment:
same here, can we move it at the bottom?

Reply:
this one I really need to execute at this point in time. Otherwise my hook in code is too late in Express. I could add two events, one at the beginning and one at the end, but I suspect we'll never use the one at the end. 
Are you ok with me keeping it at the beginning anyway? It's not as critical as the lcp stuff as well imo.

Reply:
does it make any sense caring about main thread at this stage?

Reply:
How do you mean @npeltier? All our javascript runs on the main thread and and I think the only way to move logic to another thread would be to use a web worker.

Reply:
Is the deferred event still needed after our discussion? 
My impression was that we shift loading either pre `loadArea()` or after it.